src/main/java/school/hei/haapi/model/bean/Base64EncoderBean.java

```diff
@@ -0,0 +1,13 @@
+package school.hei.haapi.model.bean;
+
+import java.util.Base64;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Base64EncoderBean {
+  @Bean
+  public static Base64.Encoder Base64Encoder() {
+    return Base64.getEncoder();
+  }
+}
```

Comment:
remove this file as we are not sure whether Base64.Encoder is Thread-safe or not